torchtitan/experiments/deepseek_v3/model.py

```diff
@@ -472,25 +468,22 @@ class MoE(nn.Module):
     # 1. "torch_all_to_all"
     # 2. "symm_mem" (see `setup_symm_mem` below)
     shuffle_method = "torch_all_to_all"
-    # Group GEMM method
-    group_mm = "torch"  # "  # [ torch, torchao, ds (ds==fp8) ]
-
-    if group_mm == "ds" and not DEEPGEMM_AVAILABLE:
-        print(
-            "DeepGemm is not available on your system.  Please install and try again."
-        )
-        raise ValueError("Missing DeepGemm install")
 
     # Symmetric memory buffers shared by all MoE instances across layers
     token_send_buf: Optional[torch.Tensor] = None
     token_gather_buf: Optional[torch.Tensor] = None
 
-    def __init__(self, config):
+    def __init__(self, config, custom_activation=None):
         super().__init__()
         self.config = config
         self.num_experts_per_tok = config.num_experts_per_tok
         # do we use triton kernel for input(activation) quantization or the default dsgemm utils (Pytorch eager based)
         self.use_triton_quant = True
+        self.rank = torch.distributed.get_rank()
```

Comment:
what is this "rank" for? If not necessary for module construction, we shall reduce runtime footprint in model description.

Reply:
it was for debugging work.  removed.